static/app/views/performance/newTraceDetails/useTraceOnLoad.tsx

```diff
@@ -167,7 +161,6 @@ export function useTraceIssuesOnLoad(
 ): 'success' | 'error' | 'pending' | 'idle' {
   const api = useApi();
   const organization = useOrganization();
-  const initializedRef = useRef<boolean>(false);
```

Comment:
If I'm reading this right, this was a mechanism to prevent making more than one request at initialization. Is that not needed anymore?

Reply:
Yeah this is not needed anymore, it was mainly there to prevent triggering the traceload analytics event unnecessarily. Decoupled the analytics from this callback in the first PR